gradle.properties

```diff
@@ -3,16 +3,16 @@ pluginGroup=fi.aalto.cs
 pluginName=A+ Courses
 pluginRepositoryUrl=https://github.com/Aalto-LeTech/aplus-courses
 # SemVer format -> https://semver.org
-pluginVersion=4.2.0
+pluginVersion=4.3.0
 # Supported build number ranges and IntelliJ Platform versions -> https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html
 pluginSinceBuild=243
-pluginUntilBuild=243.*
+# pluginUntilBuild=243.*
 # IntelliJ Platform Properties -> https://plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html#configuration-intellij-extension
 platformType=IC
-platformVersion=2024.3.1
+platformVersion=251.18673.35
 # Plugin Dependencies -> https://plugins.jetbrains.com/docs/intellij/plugin-dependencies.html
 # Example: platformPlugins = com.jetbrains.php:203.4449.22, org.intellij.scala:2023.3.27@EAP
-platformPlugins=org.intellij.scala:2024.3.18
+platformPlugins=org.intellij.scala:2025.1.3
```

Comment:
Do we now if this Scala dependency prevents it from working with IJ 2024.3?

If the course starts before 2025.1 is out we need a SBT supporting version for IJ 2024.3.

Reply:
AFAIK that version number is used only in the sandbox